tools/firmware-utils/src/dlink-sge-image.c

```diff
@@ -0,0 +1,415 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later OR MIT */
+/*
+ * This tool encrypts factory images for certain D-Link Devices
+ * manufactured by SGE / T&W, e.g. COVR-C1200, COVR-P2500, DIR-882, ...
+ *
+ * Build instructions:
+ *   gcc -lcrypto dlink-sge-image.c -o dlink-sge-image
+ *
+ * Usage:
+ *   ./dlink-sge-image infile outfile [-d: decrypt]
+ *
+ * Copyright 2021 OpenWrt.org
```

Comment:
Please add your name as the copyright holder so that it is eligible for the GNU foundation as applying to German copyright law (gpl-violations.org). OpenWrt.org is not a legal entity.

Reply:
Just found the old discussion from https://github.com/openwrt/openwrt/pull/3449, checking that again...
And from looking at other `firmware-utils`, it seems the standard is now to use `Copyright (C) YYYY Name <EMAIL>`, so I'll probably do it the same way.